ksrates/fc_plotting.py

```diff
@@ -480,18 +480,28 @@ def save_mixed_plot(fig_corr, fig_uncorr, ax_corr, ax_uncorr, species, paranome,
     :param colinearity: the config file field that states if the anchor pairs have to be plotted [True/False]
     """
     legend_size = define_legend_size(ax_corr)
-
     chart_box = ax_uncorr.get_position()
+
     # For the un-corrected plot:
-    ax_uncorr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.65, chart_box.height])
-    lgd = create_legend(ax_uncorr, paranome, colinearity, legend_size)
+    if correction_table_available:
+        ax_uncorr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.65, chart_box.height])
+        lgd = create_legend(ax_uncorr, paranome, colinearity, legend_size)
+    else:
+        lgd = ax_uncorr.legend(handlelength=1.5, mode="expand", loc="upper left", bbox_to_anchor=(0.54, 0.0, 0.75, 1))
+        ax_uncorr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.9, chart_box.height])
     fig_uncorr.savefig(os.path.join("correction_analysis", f"{species}", f"mixed_{species}_uncorrected.pdf"),
-                       bbox_extra_artists=(lgd, fig_uncorr._suptitle), bbox_inches="tight", transparent=True, format="pdf")
+        bbox_extra_artists=(ax_uncorr, lgd, fig_uncorr._suptitle), bbox_inches="tight", transparent=True, format="pdf")
+
     # Same thing for the corrected plot:
-    ax_corr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.65, chart_box.height])
-    lgd = create_legend(ax_corr, paranome, colinearity, legend_size)
-    fig_corr.savefig(os.path.join("correction_analysis", f"{species}", f"mixed_{species}_corrected.pdf"),
-                     bbox_extra_artists=(lgd, fig_corr._suptitle), bbox_inches="tight", transparent=True, format="pdf")
+    if correction_table_available:
+        ax_corr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.65, chart_box.height])
+        lgd = create_legend(ax_corr, paranome, colinearity, legend_size)
+    else:
+        lgd = ax_corr.legend(handlelength=1.5, mode="expand", loc="upper left", bbox_to_anchor=(0.54, 0.0, 0.75, 1))
+        ax_corr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.9, chart_box.height])
+
+    fig_corr.savefig(os.path.join("correction_analysis", f"{species}", f"mixed_{species}_corrected.pdf"), 
+        bbox_extra_artists=(ax_corr, lgd, fig_corr._suptitle), bbox_inches="tight", transparent=True, format="pdf")
```

Comment:
I'm not allowed to post a suggestion here directly either, but this is the new code body of this function:
```
    if correction_table_available:
        legend_size = define_legend_size(ax_corr)
        chart_box = ax_uncorr.get_position()

        ax_uncorr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.65, chart_box.height])
        lgd = create_legend(ax_uncorr, paranome, colinearity, legend_size)
        fig_uncorr.savefig(os.path.join("correction_analysis", f"{species}", f"mixed_{species}_uncorrected.pdf"),
                           bbox_extra_artists=(lgd, fig_uncorr._suptitle), bbox_inches="tight", transparent=True, 
                           format="pdf")

        ax_corr.set_position([chart_box.x0, chart_box.y0, chart_box.width*0.65, chart_box.height])
        lgd = create_legend(ax_corr, paranome, colinearity, legend_size)
        fig_corr.savefig(os.path.join("correction_analysis", f"{species}", f"mixed_{species}_corrected.pdf"),
                         bbox_extra_artists=(lgd, fig_corr._suptitle), bbox_inches="tight", transparent=True, 
                         format="pdf")
    else:
        # if not correction_table_available use a simpler layout with the legend
        # inside the plot and no right margin
        lgd = ax_uncorr.legend(handlelength=1.5, loc="upper right")
        fig_uncorr.savefig(os.path.join("correction_analysis", f"{species}", f"mixed_{species}_uncorrected.pdf"),
                           transparent=True, format="pdf")
        lgd = ax_corr.legend(handlelength=1.5, loc="upper right")
        fig_corr.savefig(os.path.join("correction_analysis", f"{species}", f"mixed_{species}_corrected.pdf"), 
                         transparent=True, format="pdf")
```